l1nk4i/docker-compose.yml

```
@@ -0,0 +1,23 @@
+version: '3.8'
```

yamllint

[error] wrong new line character: expected \n

(new-lines)